src/App.scss

```diff
@@ -2,6 +2,12 @@
 
 @import './ConstantsMixins.scss';
 
+.roverNav{
+  margin-top : 100px;
+  margin-bottom: 35px;
+  
+}
```

Comment:
Could we make these calculated values?
(ie have some shared constants so that this is always the right height)